internal/utils.go

```diff
@@ -486,3 +486,19 @@ func verifyManifestPath(baseDirectory string, manifestPath string) error {
 
 	return nil
 }
+
+// Check policy-templates to see if all the remediation actions match, if so set the root policy remediation action
```

Comment:
Using the word "set" implies a change is made to the input data, but that's not the case.
```suggestion
// Check policy-templates to see if all the remediation actions match, if so return the root policy remediation action
```

Reply:
changed.